SamplePlatformChannels/SamplePlatformChannels.csproj

```diff
@@ -42,12 +41,18 @@
 	
 	<ItemGroup Condition="$(TargetFramework.Contains('-android'))">
 		<AndroidLibrary Include="..\Platforms\Android\sample\build\outputs\aar\sample-debug.aar" Link="sample.aar" Bind="false" />
+		<ProjectReference Include="..\Microsoft.PlatformChannels.Binding\Microsoft.PlatformChannels.Binding.csproj" />
 	</ItemGroup>
-
+	<ItemGroup Condition="'$(TargetFramework)' == 'net6.0-ios'">
+		<NativeReference Include="/Users/donblas/Programming/Projects/Microsoft.Maui.Platform.Channels/Platforms/Apple/Sample/build/Debug-iphonesimulator/Sample.framework"  Kind="Framework"  />
```

Comment:
Can we do some relative paths here?

Reply:
Woops, yes. That one slipped in from debugging.

Reply:
Fixed!